### PR TITLE
style(ui): settings

### PR DESCRIPTION
**Pull request by juliendonck**
## Done
- adds settings page ui
Settings screen
- adds new `destructive` button variant
Screenshot 2020-05-12 at 15 04 16

## todo
- [x] replace `tabs` with reusable component

### CODE REVIEW

**Comment by rudolfs** [comment]
Could use some `min-width`:

Screenshot 2020-05-12 at 15 14 47

/cc https://github.com/radicle-dev/radicle-upstream/issues/304

**Comment by juliendonck** [comment]
@rudolfs fixed min-width. Will address #304 next. First version can be quite simple.

Waiting for #377 to land to merge into this and update `tabs`

**Comment by xla** [comment]
Part of #385

File: ui/App.svelte [hunk 1]
```diff
@@ -14,7 +14,7 @@
   import IdentityCreation from "./Screen/IdentityCreation.svelte";
   import DesignSystemGuide from "./Screen/DesignSystemGuide.svelte";
   import Help from "./Screen/Help.svelte";
-  import Network from "./Screen/Network.svelte";
+  import Settings from "./Screen/Settings.svelte";
```
Comment (xla) [inline ui/App.svelte:17]: Should be moved down after the `Search` import.

File: ui/Screen/Profile.svelte [hunk 2]
```diff
@@ -48,12 +44,6 @@
       href: path.profileWallet(),
       looseActiveStateMatching: false,
     },
-    {
-      icon: Icon.Settings,
-      title: "Settings",
-      href: path.profileSettings(),
```
Comment (xla) [inline ui/Screen/Profile.svelte:54]: We should delete this route in `path.ts`.

File: ui/src/path.ts [hunk 3]
```diff
@@ -45,7 +45,7 @@ export const projectSource = (
   if (revision && path) {
     return `/projects/${id}/source/${revision}/${objectType}/${
       objectType === ObjectType.Tree ? `${path}/` : path
-    }`;
+      }`;
```
Comment (xla) [inline ui/src/path.ts:48]: Is this a mistake reformatting?

File: ui/Screen/Settings.svelte [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    {
+      title: "Light",
+      value: 0,
+    },
+    {
+      title: "Dark",
+      value: 1,
+    },
+  ];
+  const networkOptions = [
```
Comment (xla) [inline ui/Screen/Settings.svelte:15]: If you want to consider options that we need for now it's:

* Emulator - for local in-memory
* FFnet - for the currently running one the team presented
* Testnet - coming soon(tm)

File: ui/Screen/Settings.svelte [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  ];
+</script>
+
+<style>
+  .container {
+    max-width: 71.25rem;
+    margin: 0 auto;
+    min-width: 640px;
+  }
+  .section-header {
```
Comment (xla) [inline ui/Screen/Settings.svelte:33]: Can we use html tags for this instead? i.e. `section header`.

File: ui/src/path.ts (hunk 3 above)
Comment (juliendonck) [reply ui/src/path.ts:48]: Yeah it's automatically putting it there on save 🤷‍♂️

File: ui/App.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/App.svelte]: 👍

File: ui/Screen/Settings.svelte (hunk 4 above)
Comment (juliendonck) [reply ui/Screen/Settings.svelte:15]: :+1:

File: ui/Screen/Settings.svelte (hunk 5 above)
Comment (juliendonck) [reply ui/Screen/Settings.svelte]: :+1:

File: ui/Screen/Profile.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/Screen/Profile.svelte:54]: :+1:

File: ui/DesignSystem/Primitive/Text.svelte [hunk 6]
```diff
@@ -22,6 +22,9 @@
     font-size: 14px;
     line-height: 20px;
   }
+  .medium {
+    font-family: var(--typeface-medium);
```
Comment (rudolfs) [inline ui/DesignSystem/Primitive/Text.svelte:26]: We should probably also add an explicit `font-size` and `line-height` if this is a new text style.

File: ui/Screen/Settings.svelte [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+            This unlinks your local repositories from Upstream. The local data
+            will remain on your computer.
+          </Text>
+        </div>
+        <div class="action">
+          <Button variant="destructive">Remove</Button>
+        </div>
+      </div>
+    </section>
+
```
Comment (rudolfs) [inline ui/Screen/Settings.svelte:158]:
```suggestion
```

File: ui/src/path.ts (hunk 3 above)
Comment (rudolfs) [reply ui/src/path.ts:47]: What's your `prettier --version`?

**Pull request merged**